src/gui/tray/UnifiedSearchInputContainer.qml

```diff
@@ -70,7 +69,23 @@ TextField {
             bottomMargin: 4
             verticalCenter: trayWindowUnifiedSearchTextField.verticalCenter
         }
+
         width: height
+
+        source: "image://svgimage-custom-color/change.svg/" + trayWindowUnifiedSearchTextField.textFieldIconsColor
+        sourceSize.width: 64
+        sourceSize.height: 64
+        rotation: 0
+        visible: trayWindowUnifiedSearchTextField.isSearchInProgress
+    }
+
+    RotationAnimator {
+        target: busyIndicator
+        running: trayWindowUnifiedSearchTextField.isSearchInProgress
+        from: 0
+        to: 360
+        loops: Animation.Infinite
+        duration: 3000
     }
```

Comment:
this should be a child of the `Image` component to directly get an hint that they are linked

Reply:
Sure, made it the child of the image